edk2toolext/invocables/edk2_platform_build.py

```diff
@@ -74,6 +74,25 @@ def RetrieveCommandLineOptions(self, args):
         """Retrieve command line options from the argparser."""
         self.PlatformBuilder.RetrievePlatformCommandLineOptions(args)
 
+    def AddParserEpilog(self):
+        """Adds additional information to the end of the argument parser."""
+        epilog = super().AddParserEpilog()
+        custom_epilog = ""
+
+        variables = self.PlatformBuilder.DeclareCriticalPlatformEnv()
+        if any(variables):
+            variables.sort(key=lambda v: v.name)
+
+            max_name_len = max(len(var.name) for var in variables)
+            max_desc_len = max(len(var.description) for var in variables)
+
+            custom_epilog += "CLI Env Variables:"
+            for v in variables:
+                custom_epilog += f"\n  {v.name:<{max_name_len}} - {v.description:<{max_desc_len}}  [{v.default}]"
```

Comment:
i like the right padding trick.

Reply:
Additionally, just added line wraps for long descriptions.